File: filmy-app/app/src/main/java/rk/entertainment/filmy/models/moviesDetails/ProductionCompanyData.java
package rk.entertainment.filmy.models.moviesDetails;

import com.google.gson.annotations.SerializedName;

public class ProductionCompanyData {

    @SerializedName("id")
    private Integer id;

    @SerializedName("logo_path")
    private String logoPath;

    @SerializedName("name")
    private String name;

    @SerializedName("origin_country")
    private String originCountry;

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getLogoPath() {
        return logoPath;
    }

    public void setLogoPath(String logoPath) {
        this.logoPath = logoPath;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getOriginCountry() {
        return originCountry;
    }

    public void setOriginCountry(String originCountry) {
        this.originCountry = originCountry;
    }

}

File: filmy-app/app/src/main/java/rk/entertainment/filmy/data/models/moviesDetails/ProductionCompanyData.kt
package rk.entertainment.filmy.data.models.moviesDetails

import com.google.gson.annotations.SerializedName

data class ProductionCompanyData(
        @SerializedName("id")
        var id: Int,

        @SerializedName("logo_path")
        var logoPath: String,

        @SerializedName("name")
        var name: String,

        @SerializedName("origin_country")
        var originCountry: String
)